docs/source/visint.rst

```diff
@@ -14,6 +41,6 @@ Sensitivities are a useful way to evaluate the tradeoffs in your model, as well
 Which produces the following plot:
 
 .. figure::  sensitivities.png
-   :width: 500 px
+   :align: center
 
-In this plot, steep lines that go up to the right are variables whose increase sharply increases (makes worse) the objective. Steep lines going down to the right are variables whose increase sharply decreases (improves) the objective. Only local sensitivities are displayed, so the lines are optimistic: the real effect of changing parameters may lead to higher costs than shown.
\ No newline at end of file
+In this plot, steep lines that go up to the right are variables whose increase sharply increases (makes worse) the objective. Steep lines going down to the right are variables whose increase sharply decreases (improves) the objective. Only local sensitivities are displayed, so the lines are optimistic: the real effect of changing parameters may lead to higher costs than shown.
```

Comment:
outside scope, but since it got edited here -- I'm wondering if someday we should do away with the sensitivity plot. I think there are better ways to show this info.